rubrics/services/pom.xml

```diff
@@ -25,56 +25,83 @@
         <rest.assured.version>2.9.0</rest.assured.version>
         <spring.security.version>4.2.1.RELEASE</spring.security.version>
         <spring.boot.version>1.5.2.RELEASE</spring.boot.version>
```

Comment:
Yeah I didn't want to push more up since these are mostly specific to this tool at this point. I only pushed what was needed in shared because of spring in shared.